src/frontend/screens/Library/index.tsx

```diff
@@ -86,15 +86,24 @@ export default React.memo(function Library(): JSX.Element {
   useLayoutEffect(() => {
     const scrollPosition = parseInt(storage?.getItem('scrollPosition') || '0')
 
-    if (listing.current !== null && scrollPosition > 0) {
-      listing.current.scrollTo(0, scrollPosition)
-    }
+    listing.current?.addEventListener('scroll', () => {
```

Comment:
would be better to extract this callback as a function like:

```ts
const storeScroll = () => {
  ...
}

listing.current?.addEventListener('scroll', storeScroll)
```

that way you can then do `listing.current?.removeEventListener('scroll', storeScroll)` without the function duplication

Reply:
done